a3p-integration/proposals/k:param-change/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "agoricProposal": {
+    "type": "/cosmos.params.v1beta1.ParameterChangeProposal"
```

Comment:
I didn't realize we had support for this type in synthetic-chain. Since we're using `eval.sh` is that actually doing what we intend?

Reply:
Yes; see [dockerfileGen.ts](https://github.com/Agoric/agoric-3-proposals/blob/f477886c6883aa6207a2f3131d01ef79034f8ca4/packages/synthetic-chain/src/cli/dockerfileGen.ts#L310) and an existing use in [61:lo-bundle-cost](https://github.com/Agoric/agoric-3-proposals/blob/main/proposals/61%3Alo-bundle-cost/package.json).